packages/gains/src/Gains.ts

```diff
@@ -0,0 +1,83 @@
+import {
+  ABI,
+  BTC_USD_PAIR,
+  CONTRACT_ADDRESS,
+  ETH,
+  ETH_USD_PAIR,
+  MARKET_ORDER_TYPE,
+  PAIRS,
+  WBTC,
+} from './constants'
+import {
+  type FilterOperator,
+  type OptionsActionParams,
+  type TransactionFilter,
+  compressJson,
+} from '@rabbitholegg/questdk'
+import { Chains, OrderType } from '@rabbitholegg/questdk-plugin-utils'
+import { type Address } from 'viem'
+
+export const options = async (
+  params: OptionsActionParams,
+): Promise<TransactionFilter> => {
+  const { recipient, amount, orderType, chainId, contractAddress, token } =
+    params
+
+  if (chainId !== Chains.ARBITRUM_ONE) {
```

Comment:
We want to retain the ability to add new chain in the future. If the chainId is not correct, the filter will not pass. We do not need to force failure.